### ISSUE TITLE
[Bug Report] Tooltips either do not open, or open and do not close in iOS safari unless it is inside a v-card

### ISSUE THREAD

**Issue by Austinchambers1** [vuetifyjs/vuetify#12067]
### Environment
**Vuetify Version:** 2.3.8
**Vue Version:** 2.6.11
**Browsers:** Safari
**OS:** iOS

### Steps to reproduce
Make a standard hover tooltip in a template with vuetify.

### Expected Behavior
It should work on tap as expected on iOS Safari

### Actual Behavior
It either does not open, or opens and does not close

### Reproduction Link
<a href="https://codepen.io/oxtailprincess/pen/XWdKpYN" target="_blank">https://codepen.io/oxtailprincess/pen/XWdKpYN</a>

**Comment by joergd** [comment]
I can confirm this happens for me too - several versions before this latest 2.3.8

**Comment by KaelWD** [comment]
Duplicate of #7114
